docarray/document/mixins/mesh.py

```diff
@@ -101,9 +101,9 @@ def load_vertices_and_faces_to_point_cloud(self: 'T', samples: int) -> 'T':
         faces = None
 
         for chunk in self.chunks:
-            if chunk.tags['name'] == MeshEnum.VERTICES:
+            if chunk.tags['name'] == MeshEnum.VERTICES.value:
```

Comment:
```suggestion
            if chunk.tags['name'] in  [MeshEnum.VERTICES.value, MeshEnum.VERTICES]
```
for sake of compatibility with previous version

Reply:
okay it was not released so not needed actually